syntax = "proto3";

option cc_generic_services = true;

message EchoRequst {
    string message = 1;
}

message EchoResponse {
    string message = 1;
}

service EchoService {
    rpc Echo(EchoRequst) returns (EchoResponse);
    rpc Echo1(EchoRequst) returns (EchoResponse);
}